src/taskpane/taskpane.html

```diff
@@ -5,16 +5,16 @@
 <html lang="en" data-framework="typescript">
 
 <head>
-    <meta charset="utf-8" />
+    <meta charset="UTF-8" />
```

Comment:
Did this come from the shared changes?
I don't think webpack is currently working for CamelCase, so that might not work

Reply:
I may have been something that's been ignored for a while, but that's the way it is in the taskpane repo and so by copying it over we avoid these little nuanced differences